store/sqlstore/shared_channel_store.go

```diff
@@ -336,11 +371,31 @@ func (s SqlSharedChannelStore) UpdateRemote(remote *model.SharedChannelRemote) (
 		return nil, err
 	}
 
-	count, err := s.GetMaster().Update(remote)
+	query, args, err := s.getQueryBuilder().Update("SharedChannelRemotes").
+		Set("Id", remote.Id).
+		Set("ChannelId", remote.ChannelId).
+		Set("CreatorId", remote.CreatorId).
+		Set("CreateAt", remote.CreateAt).
+		Set("UpdateAt", remote.UpdateAt).
+		Set("IsInviteAccepted", remote.IsInviteAccepted).
+		Set("IsInviteConfirmed", remote.IsInviteConfirmed).
+		Set("LastPostUpdateAt", remote.LastPostUpdateAt).
+		Set("LastPostId", remote.LastPostId).
+		Where(sq.Eq{"RemoteId": remote.RemoteId}).
```

Comment:
The where clause should have `id` and `channelid` as those are the primary key.

But even more importantly, this method is not used anywhere. @wiggin77 - wondering if we can remove this method? (in a separate PR of course)